src/components/10-atoms/fieldset/index.js

```diff
@@ -19,6 +19,7 @@ class AXAFieldset extends NoShadowDOM {
       horizontal: { type: Boolean, reflect: true },
       error: { type: String, reflect: true },
       slot: { type: String },
+      disableresponsivestretch: { type: Boolean, reflect: true },
```

Comment:
The "auto stretch" feature is part of "horizontal" feature. So rename that new one like "horizontal-disableresponsivestretch"

Reply:
(outdated)